beautifultable/utils.py

```diff
@@ -55,7 +60,9 @@ def get_output_str(item, detect_numerics, precision, sign_value):
 
 def ansilen(item):
```

Comment:
This method is no longer `ansilen()`, now is `east_asian_len()`.
But I could not come up with good name. 
Is there any better idea for this method.

Reply:
Don't remove the existing contents of the ansilen method. Keep the name and content intact as it has been used in many other places which could break something. Just add on top of it. Does this also work for european based unicode characters?

Reply:
>Does this also work for european based unicode characters?
It doesn't work... I'll fix it.

result:
<img width="375" alt="screen shot 2018-12-03 at 23 04 23" src="https://user-images.githubusercontent.com/1732016/49378388-cf779d80-f74f-11e8-9a5a-f9958d81e20a.png">

Reply:
I fixed this problem: 36733b5553949f55168d4ca368aa8fbd430fbe71
<img width="375" alt="screen shot 2018-12-03 at 23 38 56" src="https://user-images.githubusercontent.com/1732016/49380538-4a8f8280-f755-11e8-9dda-de9ce4da3910.png">


### Detail
European based unicode characters are treated `'A'` (Ambiguous) by `east_asian_width()`. So I used `len(ANSI_REGEX.sub('', char))` only for ambiguous character.
like this: (this is the capture before fixing) 
<img width="773" alt="screen shot 2018-12-03 at 23 21 59" src="https://user-images.githubusercontent.com/1732016/49380450-10be7c00-f755-11e8-9b64-74a3c2192ff6.png">

Reply:
Do you have something to worry about? If you have, please give me concrete examples. I may fix it 👍

Reply:
>Maybe add some comments explaining why you're doing some things

👍 

>can you just test this once with some long strings ensuring that this doesn't break when rows are wrapped.

OMG...
<img width="491" alt="screen shot 2018-12-05 at 23 34 53" src="https://user-images.githubusercontent.com/1732016/49520330-624a4080-f8e6-11e8-97e7-90a249b042fb.png">

I'll try to fix.